Jellyfin.Server.Implementations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -73,6 +74,7 @@ public static IServiceCollection AddJellyfinDbContext(
                 efCoreConfiguration = new DatabaseConfigurationOptions()
                 {
                     DatabaseType = "Jellyfin-SQLite",
+                    LockingBehavior = DatabaseLockingBehaviorTypes.NoLock
```

Comment:
```suggestion
                    LockingBehavior = DatabaseLockingBehaviorTypes.Default
```

This small naming change implies better in my opinion that there's already a locking mechanism, just that there's no override in our part.

Or...

```suggestion
                    // Use EFCore's internal locking mechanism instead of our custom one
                    LockingBehavior = DatabaseLockingBehaviorTypes.NoLock
```

Reply:
I dont know how happy i am with that. I would rather keep it as "No-Lock" and as its the first (zero) entry in the enum its implied to be the default anyway plus the comment saying its the default at most places.